modules/client/src/main/java/org/apache/ignite/internal/client/compute/ClientCompute.java

```diff
@@ -236,6 +237,37 @@ public <T, R> R executeMapReduce(TaskDescriptor<T, R> taskDescriptor, @Nullable
         return sync(executeMapReduceAsync(taskDescriptor, cancellationToken, arg));
     }
 
+    @Override
+    public <T, R> CompletableFuture<Map<ClusterNode, JobExecution<R>>> submitBroadcastPartitioned(
+            String tableName,
+            JobDescriptor<T, R> descriptor,
+            @Nullable T arg
+    ) {
+        Objects.requireNonNull(tableName);
+        Objects.requireNonNull(descriptor);
+        return failedFuture(new UnsupportedOperationException("Not implemented"));
```

Comment:
Please create a ticket and add TODO for client impl. Ideally also tickets for other clients (.NET, C++).